regform

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <title>!DOCTYPE</title>
+   <meta charset="utf-8">
+   <script>
```

Comment:
First, exclude script to a separate file and include on the page as an external resource